src/main/java/org/jitsi/videobridge/shim/ChannelShim.java

```diff
@@ -241,6 +241,26 @@ public void describe(ColibriConferenceIQ.ChannelCommon commonIq)
                 bridgeSource.setSSRC(localSsrc);
                 iq.addSource(bridgeSource);
             }
+
+            if (sources != null && !sources.isEmpty())
```

Comment:
could this not have been:
```
if (sources != null) {
    int[] ssrcs = sources.stream()
                    .map(SourcePacketExtension::getSSRC)
                    .mapToInt(Long::intValue)
                    .toArray();

    iq.setSSRCs(ssrcs);
}
```
?
I think that's a clearer

Reply:
@bbaldino Yes, it might be written in that manner, with extra `filter` invocation to ensure there is no `-1`. But I've decided here not to use stream API for that, if you insist I could do the same with stream API in different PR as long as this already merged. Thanks for review, Brian!

Reply:
Oops, yeah forgot the filter.  Ok, if I make some other changes in the bridge I may tweak that.  Thanks @mstyura